corehq/apps/app_manager/templates/app_manager/partials/releases/releases_table.html

```diff
@@ -35,7 +35,13 @@
   </p>
   <p>
     {% if request|toggle_enabled:"VIEW_APP_CHANGES" %}
-      <a class="btn btn-default" href="{% url 'app_form_summary_diff' domain latest_build_id app.id %}">
+      <a class="btn btn-default" data-bind="
+                attr: {href: reverse(
+                      'app_form_summary_diff',
+                      savedApps().length ? savedApps()[savedApps().length - 1].id() : null,
+                      '{{ app.id }}')
+                      },
```

Comment:
Does `reverse` in this context include `domain` as an argument automagically?

Also why does the `href` attr need to change now? I think what its doing is superseded by the addition of `visible`. it was also hard to wrap my head around what this was doing at first. could it be made a little clearer by adding a property that calculates this to the knockout file?

Reply:
I've made this into a property that might hopefully address some of these questions.
- Yes, `reverse` does some magic (it calls `initialPageData.reverse`)
- The `href` needs to change if the list of builds changes, since this button compares between the latest unsaved changes and the last build (i.e this is our equivalent of seeing your staged changes in git)